recipes/pybind11/all/conanfile.py

```diff
@@ -16,17 +16,25 @@ class PyBind11Conan(ConanFile):
 
     _source_subfolder = "source_subfolder"
 
+    _cmake = None
+
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
         os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
 
+    def requirements(self):
+        # self.requires("cpython/x.y/z") or self.requires("pypy/x.y/z")
+        self.output.info("This recipe requires a python SDK which is not available (yet) on CCI")
```

Comment:
there are few libraries, include this one and boost, capable of producing python modules written in C++. they usually need python development headers and libraries. we need sort of conan package to abstract it away, similar to [python-dev-config](https://github.com/bincrafters/conan-python_dev_config).